.storybook/preview.tsx

```diff
@@ -1,4 +1,4 @@
-import "@chanzuckerberg/eds-tokens/lib/css/variables.css";
+import "../lib/css/variables.css";
```

Comment:
we now read tokens from internally. if it's bad practice to read from `lib/`, can save these elsewhere in the repo